packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -792,7 +792,11 @@ const preparedCharts = computed(() => {
 			annotations
 		);
 
-		charts[variable].layer.push(...createInterventionChartMarkers(groupedInterventionOutputs.value[variable]));
+		Object.entries(groupedInterventionOutputs.value).forEach((item) => {
+			item[1].forEach((intervention) => {
+				charts[variable].layer.push(...createInterventionChartMarkers([intervention], false, -180));
+			});
+		});
 	});
```

Comment:
It looks like we are applying all the interventions to the chart. We are going to end up plotting plotting interventions for variables like I, S, and R all on the same chart. It will create a issue such that chart for variable `I`  also have intervention for other variables, for example `S` or `R` along with all interventions for parameters as well. I'm not sure if that's what we want to do.  If we want still draw all interventions for parameters for the chart, we should at least filter out the intervention for state variables (e.g. I, S, R etc) I guess.

Reply:
I think this is something @pascaleproulx needs to figure out with the design team. So until they do this, that PR can be a step forward, and we should merge it as a first version.

Reply:
Then I think we should include the interventions for the the same variable if it's for state variable, and all for parameters and filter out the intervention for other state variables. For example chart for `I` should include interventions for `I` only and plus the all interventions for the parameters (e.g `beta`).  @mloppie you can probably use `modelPartTypesMap` in line 590 to check the type to see if a variable is state variable or a parameter.

Reply:
But I still think that without proper design how we want to show parameter intervention on the state chart, just plotting all parameter interventions markers as vertical lines on the state chart won't be that useful. Parameter intervention were intentionally left out I believe. And this is going to be reworked depending on how we want to do it properly. I feel it's wise to pause until we get better idea how we should draw them on the chart from Pascale.